service/tablet_allocator.cc

```diff
@@ -1601,8 +1601,10 @@ class load_balancer {
                 if (skip) {
                     if (src_node_info.drained && skip->viable_targets.empty()) {
                         auto replicas = tmap.get_tablet_info(source_tablet.tablet).replicas;
-                        throw std::runtime_error(format("Unable to find new replica for tablet {} on {} when draining {} (nodes {}, replicas {})",
-                                                        source_tablet, src, nodes_to_drain, nodes_by_load_dst, replicas));
+                        throw std::runtime_error(
+                            "A decomission is not supported in case a keyspace replication-factor equals the number of nodes in that DC. "
+                            "Please consider reducing keyspace replication factor in order to have enough replicas"
+                        );
```

Comment:
We should preserve debug information which was there before. 

You can print "A decomission is not supported ..." only when you examined the state of topology and determined that this is really the cause of this condition. There could be other causes. E.g. removenode, not decommission. Rack overload constraints.

Reply:
@Jager227 - was this review comment addressed in newer version?